samples/client/petstore/groovy/build.gradle

```diff
@@ -19,14 +19,14 @@ buildscript {
 repositories {
     mavenCentral()
     mavenLocal()
-    mavenCentral(artifactUrls: ['http://maven.springframework.org/milestone'])
-    maven { url  "http://$artifactory:8080/artifactory/repo" }
+    mavenCentral(artifactUrls: ['https://maven.springframework.org/milestone'])
+    maven { url  "https://$artifactory:8080/artifactory/repo" }
```

Comment:
This repository is only ever going to work in very specific user environments. Is there any value in keeping it? If it's for internal corporate artifact servers, some of them are only available over HTTP (because they are only available in a companies intra-net).

I suggest either removing it or making it so that the protocol `http` vs `https` is configurable by the user.

Reply:
> Is there any value in keeping it?

I do not have an answer but I agree with you to remove these entries and only add back when there's a valid use case or reason.